web/static/css/index.css

```diff
@@ -35,12 +35,66 @@ footer {
     justify-content: center;
 }
 
+.flexboxhorizontal{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.margintop{
+    margin-top: 5px;
+    overflow: auto;
+}
+
+.margintopmaxi{
+    margin-top: 10px;
+    overflow: auto;
+}
+.portrait { 
+    border-style: solid;
+    border-color: #772b4f;
+    border-width: 2px;
+    padding: 6px;
+}
+
+.nameplayer{
+    margin-right: 5px;
+}
+
+
 main.container {
     min-height: 30rem;
+
+    background: rgba(36,36,36,1);
+    background: -moz-linear-gradient(left, 
+        rgba(36,36,36,1) 0%, 
+        rgba(31,31,31,1) 1%, 
+        rgba(31,31,31,1) 51%, 
+        rgba(31,31,31,1) 99%, 
+        rgba(36,36,36,1) 100%);
+    background: -webkit-gradient(left top, right top, color-stop(0%, rgba(36,36,36,1)), 
+        color-stop(1%, rgba(31,31,31,1)), color-stop(51%, rgba(31,31,31,1)), 
+        color-stop(99%, rgba(31,31,31,1)), color-stop(100%, rgba(36,36,36,1)));
+    background: -webkit-linear-gradient(left, rgba(36,36,36,1) 0%, 
+        rgba(31,31,31,1) 1%, 
+        rgba(31,31,31,1) 51%, 
+        rgba(31,31,31,1) 99%, 
+        rgba(36,36,36,1) 100%);
+    background: -o-linear-gradient(left, rgba(36,36,36,1) 0%, 
+        rgba(31,31,31,1) 1%, rgba(31,31,31,1) 51%, 
+        rgba(31,31,31,1) 99%, 
+        rgba(36,36,36,1) 100%);
+    background: -ms-linear-gradient(left, rgba(36,36,36,1) 0%, 
+        rgba(31,31,31,1) 1%, rgba(31,31,31,1) 51%, rgba(31,31,31,1) 99%, 
+        rgba(36,36,36,1) 100%);
+    background: linear-gradient(to right, rgba(36,36,36,1) 0%, 
+        rgba(31,31,31,1) 1%, rgba(31,31,31,1) 51%, rgba(31,31,31,1) 99%, 
+        rgba(36,36,36,1) 100%);
```

Comment:
Instead of repeat the same effect with all prefixes (`<prefix>-linear-gradient: whatever`) use just the HTML5 `linear-gradient` and then we will use CSS compiler which will auto-add this prefixes for you (this is a new issue we should create on trello).